content/Projects/Observability/RHOBS/README.md

```diff
@@ -1,5 +1,63 @@
 # Red Hat Observability Service
 
-Status: Open to internal Red Hat Teams Only.
+*Previous Documents:*
 
-TBD
+* [Initial Design for RHOBS (Internal Content)](https://docs.google.com/document/d/1cSz_ZbS35mk8Op92xhB9ijW1ivOtJuD1uAzPiBdSUqs/edit)
+* [Initial FAQ (Internal Content)](https://docs.google.com/document/d/1_xnJBS3v7n4m229L3tqCqBXzZy55yu6dxCJY-vh_Egs/edit)
+
+## What
+
+Red Hat Observability Service (RHOBS) is a managed, centralized, multi-tenancy, scalable backend for the observability data. Functionally it is an [Observatorium](../observatorium.md) as A Service (SaaS) installation. RHOBS is meant to allowing ingesting, storing and using (visualisations, import, alerting, correlation) observability signals like metrics, logging and tracing is using well established APIs.
+
+This document outlines the basic overview of the RHOBS service. If you want to learn about RHOBS architecture, look for [Observatorium](https://observatorium.io) default deployment and it's design.
+
+### Background
+
+With the amount of managed Openshift clusters, for Red Hat’s own use as well as for customers, there is a strong need to improve the observability of those clusters and of their workloads to the multi-cluster level. Moreover, with the “clusters as cattle” concept, more automation and complexity there is a strong need for a uniform service gathering observability data including metrics, logs, traces, etc into a remote, centralized location for multi-cluster aggregation and long term storage.
+
+It’s worth noting that there is also a significant benefit to collecting and using multiple signals within one system:
+* Correlating signals and creating a smooth and richer debugging UX.
+* Sharing common functionality, like rate limiting, retries, auth, etc which allows consistent integration and management experience for users
+
+The Openshift Monitoring Team began preparing for this shift in 2018 with the [Telemeter](telemeter.md) Service. In particular, while creating the second version of the Telemeter Service, we put effort into developing and contributing to open source systems and integration to design the [“Observatorium”](../observatorium.md): a multi-signal, multi-tenant system capable that can be operated easily and cheaply as a SaaS either by Red Hat or on-premise. After extednding scope of the RHOBS, Telemeter become the first "tenant" of the Red Hat Observability Service.
+
+In Summer 2020, the Monitoring Team together with the Openshift Logging Team added a logging signal to the [“Observatorium”](../observatorium.md) and started to manage it for internal team as the Red Hat Observability Service.
```

Comment:
I think we need to be consistent in our usage of either the abbreviation or the full name
```suggestion
In Summer 2020, the Monitoring Team together with the Openshift Logging Team added a logging signal to the [“Observatorium”](../observatorium.md) and started to manage it for internal team as the RHOBS.
```